broker-util/oo-admin-ctl-app

```diff
@@ -217,6 +217,15 @@ begin
       else
         cis = g.component_instances
         cis.each do |ci|
+          app.group_overrides.each do |ago|
+            unless ago.nil?
+              if !ago.min_gears.nil? && ci.gears.length <= ago.min_gears
+                puts "Gear #{gear_uuid} cannot be removed as the application " +
+                "already reaches the minimum gears limit which is #{ago.min_gears}."
```

Comment:
Do you mind if we change the wording here a bit? It does get the point across, but perhaps stating it something like the below will be a bit more clear:
```
"has already reached the minimum gear limit of #{ago.min_gears}."
```